rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -644,57 +644,64 @@ def find_recursive_deployments(self, source_id):
                         visited[dependency.target_deployment_id] = True
         return results
 
-    def increase_deployment_counts_in_graph(self, target, source):
+    def increase_deployment_counts_in_graph(self, target_id, source):
         """
         Increase the deployment counts for target deployment that is
         referenced directly by the `source` deployment and then make sure to
         propagate that increase to all deployment that are referenced
         directly and indirectly by `target` deployment
-        :param target: Target Deployment node that we need to attach source to
-        :rtype Deployment
+        :param target_id: Target Deployment ID that we need to attach
+        source to
+        :rtype str
         :param source: Source deployment that reference target deployment
         :rtype Deployment
         """
-        total_services = source.sub_services_count or 0
```

Comment:
This (and the following line's change) seems to just make some of the following code more complicated- is there any reason to take this approach?

Reply:
@geokala When I checked the default values for integer counter fields in all our models I see that default value is not zero so I sticked with the same approach and the reason why I'm putting this `or` is to help in summation below and prevent adding NoneType to integer. But I think I;m going to change the default value for all counters for deployment to fallback to zero instead which will make this code more cleaner